code-challenges.js

```diff
@@ -10,33 +10,87 @@ var temp1 = 35
 
-
+// Declare a function that takes in a number value, representing temperature, as an argument
+const boilingPoint = (num) => {
+  // Use if conditional to compare if number is less than boiling point
+  if (num < 212) {
+    return `${num} is below boiling point.`
+    // Use else if conditional if number is greater than boiling point
+  } else if (num > 212) {
+    return `${num} is above boiling point.`
+    // Use else if condiional if number is at boiling point
+  } else if (num === 212){
+    return `${num} is at boiling point.`
+    // Use else as error catch all
+  } else {
+    return "Please enter a number"
+  }
+}
+// Run variables through function to test
+console.log(boilingPoint(temp1))
+console.log(boilingPoint(temp2))
+console.log(boilingPoint(temp3))
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
+// Declare a function that takes in an array as an argument
+const multFive = (arr) => {
+  // Declare local array variable to take in the new array
+  let multFiveArray = []
+  // Use a for loop to iterate through each index of the array
+  for(let i = 0; i < arr.length; i++) {
+     // Take value at each index and multiply by 5
+     multFiveArray.push(arr[i] * 5)
+   }
+   // Return new array
+   return multFiveArray
+}
 
+//Run global array through function to test
+console.log(multFive(myNumbers1))
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
+// Declare a function that takes in an array as an argument
+const multFiveMap = (arr) => {
+  // Return and map through array and multiply each value by 5
+  return arr.map(value => value * 5)
+}
 
-
+// Run array through function to test
+console.log(multFiveMap(myNumbers2))
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
+// Declare a function that takes in a string as an argument
+const noVowels = (str) => {
+  // Declare a local variable array that stores each vowel at a new index
+  let vowels = ["a", "e", "i", "o", "u"]
+  // Return an array as variable that will split the string's characters into individual indexes
+  let strArray = str.split("")
+  // Return filtered array through filtering through the above new array
+  return filteredArray = strArray.filter(value => {
+    // Return only values where an index number exist within the vowels array using .indexOf (i.e. index # less than 0)
+    return vowels.indexOf(value) < 0
+    // Join back to string through .join
+  }).join("")
+}
```

Comment:
this looks great! your solution may have more syntax than you need, even! also, it does not account for uppercase vowels. here's a refactored version of your code (using the exact same logic + a way to deal with cases) to consider:

```javascript
const noVowels = (str) => {
  // create a variable to house vowel values
  let vowels = ["a", "e", "i", "o", "u"]
  // split array, filter it (change values to lowercase, then join back)
  return str.split("").filter(value => vowels.indexOf(value.toLowerCase()) < 0).join("")
}
```